src/main/java/be/seeseemelk/mockbukkit/WorldMock.java

```diff
@@ -826,18 +824,27 @@ public void save()
 	}
 
 	@Override
-	public <T extends Entity> T spawn(Location location, Class<T> clazz) throws IllegalArgumentException
+	public <T extends Entity> T spawn(@NotNull Location location, @NotNull Class<T> clazz) throws IllegalArgumentException
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		Validate.notNull(location, "The provided location must not be null.");
+		Validate.notNull(location, "The provided clazz must not be null.");
```

Comment:
```suggestion
		Validate.notNull(clazz, "The provided class must not be null.");
```

Reply:
Thank you. After copying, I forgot to change it. I fixed in [`a82c24a` (#269)](https://github.com/MockBukkit/MockBukkit/pull/269/commits/a82c24ae663366a86473a152e0fde069c4e8e836), but I didn't fix the message because I think it is better the same as the argument name.